src/components/PipelineResourcesModal/UniversalFields.js

```diff
@@ -0,0 +1,84 @@
+/*
+Copyright 2019 The Tekton Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import React from 'react';
+import { Dropdown, TextInput } from 'carbon-components-react';
+import NamespacesDropdown from '../../containers/NamespacesDropdown';
+
+const itemToString = item => (item ? item.text : '');
+
+const UniversalFields = props => {
+  const {
+    name,
+    handleChangeTextInput,
+    handleChangeNamespace,
+    selectedNamespace,
+    type,
+    handleChangeType,
+    url,
+    invalidFields
+  } = props;
+
+  return (
+    <>
+      <TextInput
+        id="name"
+        placeholder="pipeline-resource-name"
+        value={name}
+        labelText="Name"
+        onChange={handleChangeTextInput}
+        invalid={'name' in invalidFields}
+        invalidText="Must be less than 64 characters and contain only lowercase alphanumeric characters or -"
+        autoComplete="off"
+      />
+      <NamespacesDropdown
+        id="namespace"
+        selectedItem={
+          selectedNamespace
+            ? {
+                id: selectedNamespace,
+                text: selectedNamespace
+              }
+            : ''
+        }
+        onChange={handleChangeNamespace}
+        invalid={'namespace' in invalidFields}
+        invalidText="Namespace required."
+      />
+      <Dropdown
+        id="type"
+        titleText="Type"
+        label=""
+        initialSelectedItem={{
+          id: type,
+          text: type === 'Git' ? 'Git' : 'Image'
+        }}
+        items={[{ id: 'git', text: 'Git' }, { id: 'image', text: 'Image' }]}
+        itemToString={itemToString}
+        onChange={handleChangeType}
+      />
+      <TextInput
+        id="url"
+        placeholder="pipeline-resource-url"
+        value={url}
+        labelText="URL"
+        onChange={handleChangeTextInput}
+        invalid={'url' in invalidFields}
+        invalidText="URL required"
+        autoComplete="off"
```

Comment:
Why off?

Reply:
idk, just copied it from the universal fields for secrets which all had auto-complete off